source/summary.rst

```diff
@@ -40,41 +40,40 @@ funded through the EU Horizon 2020 programme.
 Attack model and terminology
 ++++++++++++++++++++++++++++
 
-We consider a *network adversary* that can read, modify, and create
-network messages.
-Examples of such an adversary are an ISP, an e-mail provider, an AS,
-or an eavesdropper on a wireless network.
-The goal of the adversary is to i) read the content of messages, ii)
-impersonate peers -- communication partners, and iii) to learn who communicates
-with whom.
-To achieve these goals,
-an active adversary might try, for example,
-to perform a machine-in-the-middle attack on the key exchange protocol
-between peers.
-
-To enable secure key-exchange and key-verification between peers,
-we assume that peers have access to a *out-of-band*
-communication channel that cannot be observed or manipulated by the adversary.
-More concretely we expect them to be able
-to transfer a small amount of data via a QR-code confidentially.
-
-Targeted attacks on end devices or the out-of-band channels
-can break our assumptions
-and therefore the security properties of the protocols described.
-In particular
-the ability to observe QR-codes in the scan process
-(for example through CCTV or by getting access to print outs)
-will allow impersonation attacks.
-Additional measures can
-relax the security requirements for the *out-of-band* channel
-to also work under a threat of observation.
+We consider
+
+- *Peer* devices that use the network or *first channel* for transporting messages
+  and for key-exchange in order to establish end-to-end encryption.
+
+- A *network adversary* that can read, modify, and create
+  network messages on the *first channel*.
```

Comment:
It took me a while to understand what "first channel" means (it simply is the normal channel, i.e. messages sent via email, and is supposed to be the opposite of the "second channel") - I guess that the confusion arised because the word "second channel" is established at some other point, so here it is surprising that there is an extra word for something as simple as the "normal" channel.

Of course I'm biased because I read the document in the previous form very often, so I can't say whether it was better. But for me personally, only using the word "second channel" or "out-of-band communication channel", while using "network messages" instead of "first channel" seems more intuitive.

Reply:
k, i stroke "first channel" and re-introduced the previous wording.